models/UserSearch.php

```diff
@@ -95,6 +99,7 @@ public function search($params)
 
         $query->andFilterWhere(['like', 'username', $this->username])
             ->andFilterWhere(['like', 'email', $this->email])
+            ->andFilterWhere(['user.id' => $this->id])
```

Comment:
Hi, pls. even replace user.id with {{%user}}.id so if a table prefix is set, it will be applied